cms/templates/flexiblepricing/flexible_pricing_request_form_verification.html

```diff
@@ -5,21 +5,12 @@
 
 {% block content %}
 
-  <div id="main" class="resource-banner">
-    <div class="container">
-      <h1 class="text-uppercase">{{ page.title }}</h1>
-      {% if page.sub_heading %}
-        <h2>{{ page.sub_heading }}</h2>
-      {% endif %}
-    </div>
-  </div>
-
 <div class="container resource-container">
   <div class="row">
     <div class="col-md-2">
     </div>
     <div class="col-md-8">
-        <h2 class="text-uppercase" id="{{ page.title|slugify }}">{{ page.title }}</h2>
+        <h1 class="text-uppercase" id="{{ page.title|slugify }}">{{ page.title }}</h1>
 
         <p>Before you can pay, you need to verify your income. Please visit the 
           <a href="https://na2.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=4a74536d-1629-4709-b8e9-f173a51cf501&env=na2&v=2">secure DocuSign website</a>
```

Comment:
Thinking this bit should be displayed along with the thank you text and on the status display - not sure this template gets called ever, actually, so it can really go away.

Reply:
Yes. The link will be specific to DEDP.

Reply:
Is there a place in the CMS where we can customize the message when the user needs review? This is where we would put the link to docusign.

Reply:
There's fields in the CMS to customize the text displayed after the submission is submitted and when the request is waiting to be processed. So, that link can just be added there and this template itself can be removed.